iceoryx_posh/source/popo/ports/interface_port.cpp

```diff
@@ -38,6 +38,7 @@ void InterfacePort::dispatchCaProMessage(const capro::CaproMessage& caProMessage
         // information loss for this interface port
         errorHandler(Error::kPOSH__INTERFACEPORT_CAPRO_MESSAGE_DISMISSED, nullptr, ErrorLevel::SEVERE);
     }
+    /// @todo notify the condition variable here
```

Comment:
Please no todos in the code.

Reply:
See [this](https://github.com/eclipse-iceoryx/iceoryx/pull/784#discussion_r633843244) comment.